build.gradle

```diff
@@ -86,6 +86,9 @@ dependencies {
 
     implementation group: 'org.luaj', name: 'luaj-jse', version: '3.0.1'
 
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
+    implementation group:  'org.tinspin', name : 'tinspin-indexes', version: '1.7.1'
```

Comment:
This seems unused? Can't find any imports of `org.tinspin`.

Reply:
I used the phtree inside the library to query the neighbor entities of player, now I extracted it from org.tinspin